aspnetcore/security/enforcing-ssl.md

```diff
@@ -363,7 +363,7 @@ See [Configure trust of HTTPS certificate using Firefox browser](#trust-ff-ba) i
 
 ### Trust the certificate with Fedora 34
 
-See [this GitHub comment](https://github.com/dotnet/aspnetcore/issues/32361#issuecomment-837111639).
+See [Set up a .NET development environment](https://fedoramagazine.org/set-up-a-net-development-environment/) on Fedora and [this GitHub comment](https://github.com/dotnet/aspnetcore/issues/32361#issuecomment-837111639).
```

Comment:
```suggestion
See [this GitHub comment](https://github.com/dotnet/aspnetcore/issues/32361#issuecomment-837111639) and [Set up a .NET development environment](https://fedoramagazine.org/set-up-a-net-development-environment/) on Fedora.
```

Reply:
It might be helpful for a deeper understanding of how fedora uses the certificates. 

https://docs.fedoraproject.org/en-US/quick-docs/using-shared-system-certificates/

Reply:
> It might be helpful for a deeper understanding of how fedora uses the certificates.
> 
> https://docs.fedoraproject.org/en-US/quick-docs/using-shared-system-certificates/

Thanks @omarerc , see #28049